README.md

```diff
@@ -10,10 +10,24 @@ Generally speaking, the role defined in this module should also map 1:1 to that
 
 ## Usage
 
+In most cases, you will just use the `source_account_id` parameter to trust the user and group managment account; you can then keep all management of which of those users and groups can assume roles there. The following code illustrates that pattern:
+
+```hcl
+module "aws_iam_dest_user_group_role" {
+  source  = "trussworks/iam-cross-acct-dest/aws"
+  version = "1.0.3"
+
+  iam_role_name     = "group-name"
+  source_account_id = "account-id"
+}
+```
+
+However, if you want to make the dependency on the source role explicit, you can do it by adding the `source_account_role_names` parameter, like the following example:
```

Comment:
Would it be helpful to add something about how this method is utilizing IAM role chaining and maybe linking to the role chaining section of the AWS docs -- https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html